recipes-graphics/mesa/tegra-udrm-gbm_1.0.2.bb

```diff
@@ -14,6 +14,8 @@ SRC_URI = "git://${SRC_REPO};branch=${SRCBRANCH}"
 # v1.0.2 tag
 SRCREV = "103d33d07e6c38106bf952403b336dc3ca809a08"
 
+SRC_URI:append:tegra234 = " file://0001-update-backend-and-library-name.patch"
```

Comment:
I would like to know if there's a clean way to update the original library to support both tegra-udrm and nvidia-drm, rather than patching this

Reply:
I'm trying another alternative. I will push updates once I'm done with the testing

Reply:
I tried with a symlink creation and it works fine. We don't need the patch